cfme/tests/automate/test_class.py

```diff
@@ -169,3 +170,34 @@ def test_class_display_name_unset_from_ui(get_namespace):
     with update(a_class):
         a_class.display_name = ""
     assert a_class.exists
+
+
+@test_requirements.automate
```

Comment:
No need to put this, requirement is already satisfied by pytestmark

Reply:
Thanks @valaparthvi